app-text/xpdf/xpdf-4.04-r1.ebuild

```diff
@@ -92,6 +91,11 @@ src_configure() {
 		-DWITH_LIBPNG=$(usex png)
 		-DXPDFWIDGET_PRINTING=$(usex cups)
 		-DSYSTEM_XPDFRC="${EPREFIX}/etc/xpdfrc"
+		# Bug 910166
+		# https://forum.xpdfreader.com/viewtopic.php?p=45052#p45052
+		# Likely fixed in 4.05
+		-DCMAKE_DISABLE_FIND_PACKAGE_Qt6Widgets=ON
```

Comment:
Do ON for both please to be consistent

Reply:
It's mentioned like this in the `INSTALL` file. The thought was to allow this to be found if need be. Updated.